x/ecocredit/server/msg_server.go

```diff
@@ -208,18 +204,17 @@ func (s serverImpl) Send(goCtx context.Context, req *ecocredit.MsgSend) (*ecocre
 			return nil, err
 		}
 
-		tradable, err := math.ParseNonNegativeFixedDecimal(credit.TradableAmount, maxDecimalPlaces)
+		tradable, err := math.NewPositiveFixedDecFromString(credit.TradableAmount, maxDecimalPlaces)
```

Comment:
This needs to be `NonNegative`, because it should allow zeroes

Reply:
[fixed](https://github.com/regen-network/regen-ledger/pull/452/commits/f72b7d629550f348ceb8e0262b822b353df5a709#diff-f46ab4cd5ad5a92a9e0ba24bd4a5108de88dcb3e5f394835cce9001b9f683600R207)